Coder Desktop/VPN/PacketTunnelProvider.swift

```diff
@@ -1,12 +1,17 @@
 import NetworkExtension
+import os
 
 class PacketTunnelProvider: NEPacketTunnelProvider {
+    private var logger = Logger(subsystem: "com.coder.Coder-Desktop.VPN", category: "provider")
```

Comment:
```suggestion
    private var logger = Logger(subsystem: Bundle.main.bundleIdentifier!, category: "provider")
```
After the changes in my PR are merged, these will be the same value.